_docs-v5/upgrading-from-v4.md

```diff
@@ -1529,6 +1530,40 @@ In list view, the "No events to display" message.
   </tr>
 </table>
 
+## Event Data
+
+<table>
+  <tr>
+    <td>
+      <ul class='diff-list'>
+        <li class='diff-removed'>duration</li>
+      </ul>
+    </td>
+<td markdown='1'>
+
+In v4, the events using duration property were added to event.extendedProps and was accesible from there, however in v5 this property is now an Event class property and will not be accesible from event.extendedProps anymore, a workaround would be to provide an alternative name like in [this example](https://codepen.io/LiamBull/pen/mdVpwme)
```

Comment:
could you remove the codepen link? i like to keep all runnable examples within the codebase. and a runnable demo is overkill in this scenario

Reply:
makes sense, done